report_thesis/src/sections/results/initial_results.tex

```diff
@@ -0,0 +1,93 @@
+\subsection{Initial Results}
+As described in Section~\ref{sec:proposed_approach}, we conducted a series of initial experiments to evaluate the performance of various machine learning models on the prediction of major oxide compositions from our \gls{libs} dataset.
+These experiments aimed to provide a preliminary assessment of the models' performance, allowing us to identify the most promising models for further evaluation and inclusion in our stacking ensemble.
+All models were trained on the same preprocessed data using the Norm 3 preprocessing method described in Section~\ref{sec:norm3}.
+This ensured that the models' performance could be evaluated under consistent and comparable conditions.
+
+Table~\ref{tab:init_results} presents the results of these experiments, including the \gls{rmsep}, \gls{rmsecv}, standard deviation, and standard deviation of cross-validation prediction errors for each model across all oxides.
+The means of each metric are also provided to give an overall indication of the models' performance.
+Furthermore, we present an overview of these mean values in Figure~\ref{fig:init_results_rmses} to facilitate a visual comparison of the models' general performance.
+
+The results indicate that the gradient boosting models, \gls{xgboost}, \gls{gbr}, and \gls{ngboost}, consistently perform well across all oxides, with \gls{xgboost} generally outperforming the other two gradient boosting models.
+These models exhibit both low mean \gls{rmsep} and \gls{rmsecv} values, indicating high accuracy, as well as low standard deviation values, underscoring their robustness.
+\gls{svr} is also among the top-performing models, with mean \gls{rmsep} and \gls{rmsecv} values close to those of \gls{xgboost} and low standard deviation values.
+
+While usually outperformed by gradient boosting models and \gls{svr}, the ensemble models, \gls{rf} and \gls{etr}, also exhibit good performance.
```

Comment:
ahh we need to be careful, i just remembered an issue in the model selection section i forgot to point out a few days ago

gradient boosting models are also ensemble models

(issue in table is: we call the linear models 'regression' models - they all are! I'll fix this)

Reply:
so can you correct your use of `ensemble` here?

Reply:
here's what I mean

![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/33ce2add-4353-444e-be3d-7dec3e372df9)

Reply:
Unsure what term you want me to use then, could you make a suggestion to ensure it aligns with the table fix?

Reply:
Yeah, the table thing was just a tangent because I remembered it and wanted to let you know.

The issue here is that it sounds like we don't recognize the GB models as ensembles, which they are.
Just do something like
```suggestion
While usually outperformed by gradient boosting models and \gls{svr}, other ensemble models, such as \gls{rf} and \gls{etr}, also exhibit good performance.
```

Reply:
But probably not `such as`

Reply:
```suggestion
While usually outperformed by gradient boosting models and \gls{svr}, other ensemble models, including \gls{rf} and \gls{etr}, also exhibit good performance
```